generate_diff.py

```diff
@@ -1,12 +0,0 @@
-from regparser.diff import api_reader
```

Comment:
Hah, it's the little scripts like this one that kept me from running everything.